MTPot.py

```diff
@@ -119,6 +188,18 @@ def handleException(self, exc_type, exc_param, exc_tb):
         honey_logger.debug(traceback.format_exc())
         return True
 
+    #Catch tracestack error
+    def inputcooker(self):
+        try:
+            TelnetHandler.inputcooker(self)
```

Comment:
You are correct, so I think we should replace the class deceleration to - class MyTelnetHandler(TelnetHandler, object) and change this line to super(MyTelnetHandler, self).inputcooker()